index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Login</title>
+  </head>
+  <body>
+    <div class="login-session">
+      <h2>Login</h2>
+      <h3>Enter your credentials</h3>
+      <a href="./register.html" id="register-html">Register here</a>
+      <form onsubmit="return false;" class="login-form">
+        <select id="select" required>
+          <option value="accountNumber" text="accountName">
```

Comment:
The text should be a meaningful English word i.e. "Account Number" instead of "accountNumber". The text is what the user sees, the value is what the computer understands.

Reply:
I believe this should be the default option. The value should be an empty string i.e. value=""